orders/models.py

```diff
@@ -1,3 +1,24 @@
 from django.db import models
+from customer.models import Customer
 
-# Create your models here.
+
+class Orders(models.Model):
+    customer = models.ForeignKey(Customer, on_delete=models.CASCADE)
+    order_notes = models.TextField(blank=True, null=True)
```

Comment:
is there a specific reason why a textField insted of charField

Reply:
no

Reply:
did u change it then ?

Reply:
no